projects/vdk-core/tests/functional/ingestion/test_ingest_chain.py

```diff
@@ -9,68 +9,63 @@
 from unittest import mock
 
 from click.testing import Result
-from functional.ingestion import utils
-from functional.ingestion.payload_pre_process_plugins import AddPayloadSizeAsColumn
-from functional.ingestion.payload_pre_process_plugins import (
-    ConvertPayloadValuesToString,
-)
+from vdk.plugin.test_utils.ingest_util_plugins import AddPayloadSizeAsColumn
+from vdk.plugin.test_utils.ingest_util_plugins import ConvertPayloadValuesToString
+from vdk.plugin.test_utils.ingest_util_plugins import DummyIngestionPlugin
 from vdk.plugin.test_utils.util_funcs import cli_assert_equal
 from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import jobs_path_from_caller_directory
 from vdk.plugin.test_utils.util_plugins import IngestIntoMemoryPlugin
 
-# TODO: Add test cases for the following scenarios
-# 1) INGEST_PAYLOAD_PREPROCESS_SEQUENCE=A,B in job code method = C
-# 2) INGEST_PAYLOAD_PREPROCESS_SEQUENCE=A,B and VDK_INGEST_METHOD_DEFAULT=C
-# 3) INGEST_PAYLOAD_PREPROCESS_SEQUENCE=A,B and VDK_INGEST_METHOD_DEFAULT=C and
-#    job code is D
-# 4) INGEST_PAYLOAD_PREPROCESS_SEQUENCE=""
-# 5) INGEST_PAYLOAD_PREPROCESS_SEQUENCE=A,B - A,B are used for pre-processing,
-#    B is used for ingesting and post-processing
-
 
 log = logging.getLogger(__name__)
 
 
+# INGEST_PAYLOAD_PREPROCESS_SEQUENCE=A,B and VDK_INGEST_METHOD_DEFAULT=C
```

Comment:
I think test methods need to be sorted by complexity, starting with ingestion scenarios only (no processing), then pre-processing, then post-processing, then entire end-to-end verification

Reply:
Tests sorted. Thanks